src/components/pages/chefProfile/mealsTab/ChefProfilePageMealsTab.tsx

```diff
@@ -34,12 +45,16 @@ export default function ChefProfilePageMealsTab({ cookId }: ChefProfilePageMeals
                     style={{ alignItems: 'center' }}
                 >
                     <HStack className="overflow-x-auto w-[100%-80px] gap-2" style={{ justifyContent: 'flex-start' }}>
-                        <PETabItem title="All" onClick={(): void => setSelectedTab(undefined)} active={selectedTab === undefined} />
+                        <PETabItem
+                            title={t('label-all')}
+                            onClick={(): void => setSelectedTab(undefined)}
+                            active={selectedTab === undefined}
+                        />
 
                         {mealTypes.map((mealType, index) => (
                             <PETabItem
                                 key={index}
-                                title={mealType.toLowerCase()}
+                                title={mealTypesTranslates?.[mealType] ?? mealType.toLowerCase()}
```

Comment:
mealTypesTranslates is for sure not undefined here, so the optional chaining is useless I think:
```suggestion
                                title={mealTypesTranslates[mealType] ?? mealType.toLowerCase()}
```